Fix ztrans2.pl

jeremyirilli opened the pull request:
It would throw an exception: 

![grafik](https://github.com/user-attachments/assets/ca8bda71-8180-4e78-951e-72f724df1821)


although there is nothing wrong with the expression. Instead, it is interval(available(R0), true) that failed, because the result is 1.5NaN, but this is normal and should not be captured by the exception.

After:
![grafik](https://github.com/user-attachments/assets/6271d71d-3be4-4609-a333-5b69c3d20245)

mgondan commented:
I don’t understand the PR, what exactly did you change?

if NaN is a problem, don’t we need to fix it in interval?

jeremyirilli commented:
First I added a missing hook in ztrans2.pl and then I changed search.pl so that it does not throw an expection when the result is NaN. That exception (that you added recently) should only be thrown when a function or hook is missing, not when the result is NaN.
